### ISSUE TITLE
Automatic replacement of asterisks (*) to bullets in codebox

### ISSUE THREAD

**Issue by philippHRO** [giuspen/cherrytree#293]
Cherry Tree replaces leading asterisks (*) to bullet points as a help to create nice looking lists. This is fine, but it also happens in code boxes. 
The problem here is, that some languages have the asterisk as command to start a comment, in my case it's IBM's SPSS, a statistical program.

So after Cherry Tree replaced the asterisks, my code won't run in SPSS...

Note that this behaviour is completely independant of the automatic syntax highlighting feature. The replacement always happens, even with highlighting turned off.

TL;DR: 
Cherry Tree needs to stop replacing leading asterisks in code boxes.

**Comment by giuspen** [comment]
in this case, are you using a code box with plain text or with a syntax highlighting?

**Comment by philippHRO** [comment]
It happens with plain text codeboxes and hsyntax highlighting codeboxes.

**Comment by giuspen** [comment]
Ok but what are you using

**Comment by philippHRO** [comment]
Sorry, I don't understand the question. I tested both cases, and both show the behavior.

**Comment by giuspen** [comment]
What do you use for IBM's SPSS

**Comment by philippHRO** [comment]
I kinda stuck with the style for "R", since there is none for SPSS specifically.
Note that I'm not actually using the "execute codebox" function. i just store the scripts and copy/paste them from & into spss manually.

**Comment by giuspen** [comment]
I'm thinking to disable for the code and leave enabled for plain and rich text. About the missing syntax highlighting for SPSS, I found one guy implemented it https://github.com/mescobal/cussynhig
You have to place sps.lang in the cherrytree *language-specs* folder to try it. If it works fine I can include it in cherrytree. let me know if you need further help to try it.

**Comment by philippHRO** [comment]
I tried it.
SPS does show up in the list, but when I choose it and hit ok, the highlighting is just normal text, no colors.

**Comment by giuspen** [comment]
I see. I cannot work on sps.lang myself but if you have time and want to improve it, the syntax is described in https://developer.gnome.org/gtksourceview/stable/lang-tutorial.html About this issue I confirm I will disable the auto replacement for code but leave it for plain text
